src/protocols/mcsp/MessageCounterManager.h

```diff
@@ -24,24 +24,41 @@
 
 #include <protocols/Protocols.h>
 #include <system/SystemPacketBuffer.h>
+#include <transport/MessageCounterManagerInterface.h>
 
 namespace chip {
-namespace Messaging {
-
-constexpr uint16_t kMsgCounterSyncRespMsgSize = 12;  // The size of the message counter synchronization response message.
-constexpr uint32_t kMsgCounterSyncTimeout     = 500; // The amount of time(in milliseconds) which a peer is given to respond
-                                                     // to a message counter synchronization request.
+namespace mcsp {
 
 class ExchangeManager;
 
-class MessageCounterSyncMgr : public Messaging::ExchangeDelegate
+class MessageCounterManager : public Messaging::ExchangeDelegate, public Transport::MessageCounterManagerInterface
 {
 public:
-    MessageCounterSyncMgr() : mExchangeMgr(nullptr) {}
+    static constexpr uint16_t kChallengeSize = Transport::PeerMessageCounter::kChallengeSize;
+    static constexpr uint16_t kCounterSize   = 4;
+    static constexpr uint16_t kSyncRespMsgSize =
+        kChallengeSize + kCounterSize; // The size of the message counter synchronization response message.
+    static constexpr uint32_t kSyncTimeout =
```

Comment:
add unit of measurement to constants: use `kSyncTimeoutMs` so your comment does not need to specify the unit (and when reading code I do not need to read comment to know units)

Reply:
done